src/test/java/HW17.java

```diff
@@ -0,0 +1,22 @@
+
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class HW17 extends MyBaseTest {
+
+
+ @Test
+ public void addASongToPlaylist(){
+
+  navigateToPage();
+  login("zahinedu@gmail.com","te$t$tudent");
+  searchSong("take my hand");
+  clickviewAll ();
+  addField ();
+  addToField();
+  verifyAddField ();
+  songField();
+}
+}
```

Comment:
**How can I beautify code in IntelliJ:**
The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.